.github/workflows/release-v2.yml

```diff
@@ -0,0 +1,47 @@
+name: Release v2 version
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/core
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run Setup
+        uses: ./.github/actions/setup
+        with:
+          npm_token: ${{ secrets.npm_token }}
+      - uses: ./.github/actions/git-creds
+      - name: Check current branch is "version2"
+        run: |
+          if [[ "$(git branch --show-current)" != "version2" ]]; then
+            echo "Error: Not on 'version2' branch."
+            exit 1
+          fi
+      - name: Build
+        run: yarn build
+      - name: Lint
+        run: yarn lint
+      - name: Test
+        run: yarn test
+      - name: Stylelint
+        run: yarn stylelint
```

Comment:
why don't you use the `build-and-upload` and `test` jobs before running `release` job instead? as it is in the release.yml now

Reply:
oh they're dependent on lerna...